arangod/Cluster/ClusterComm.h

```diff
@@ -288,6 +293,15 @@ struct ClusterCommResult {
     // result was available in different status code situations :S
     if (single) {
       status = result->wasHttpError() ? CL_COMM_ERROR : CL_COMM_SENT;
+      if (status == CL_COMM_ERROR) {
+        try {
```

Comment:
Should the error code be initialized to something else than `TRI_ERROR_NO_ERROR`?